pull_request_template.md

```diff
@@ -0,0 +1,45 @@
+# Description
+
+Please include a summary of the changes and the related issue. Please also include relevant motivation and context. List any dependencies that are required for this change. Include if this was a client requested change. Include reference to original request.
+
+
+## Related Issues
+
+Link to any GitHub issues or client requests being addressed in this PR (use "#"):
```

Comment:
Do you want every PR to have a ticket attached? I would have them write an issue if they contribute if there isn't already one anyway. But let me know if there is a reason this could be optional. For instance I would word it more directly as

##

Link to relevant GitHub issues or client requests being addressed in this PR (use "#"). If no issue exists create it on `<Project-Board-Name>`

Something like the above text to make it clear it is required. Because it is easy to track especially as to why they would have even made code in the first place

Reply:
Is the extra step of creating the issue necessary if the description addresses why the code was made? Like if this is a PR for a feat that doesn't have an associated issue, do we still need to create an issue?

Would it be sufficient to change this to something like:

Link to any GitHub issues or client requests being addressed in this PR (use "#"), if no issue exists please include a statement as to why this code is necessary.

Reply:
It is more of a failsafe for you guys if new people come in they give you reasoning why they even started on it. It more gives them and you more thought as to why they actually did it. Did they plan it out with research. Did they really have a reason? 

Why are they even making a PR or changes if it wasn't an issue to begin with?

That is more my thought process.

I can make a lot of changes for the hell of it. But why am I?

Does that make sense? 🤔